packages/web-components/fast-components-msft/src/divider/fixtures/divider.html

```diff
@@ -0,0 +1,8 @@
+<fast-design-system-provider use-defaults>
+    <h1>Divider</h1>
+    <h4>Default</h4>
+    <fast-divider></fast-divider>
+
+    <h4>Presentation</h4>
+    <fast-divider role="presentation"></fast-divider>
```

Comment:
@microsoft/fast-dna-collaborators We've talked about having two different recipes based on whether this needs to be accessible to a screen reader. In the event that the divider is only for presentation, the recipe would not need to meet contrast requirements. In the event that it is exposed to AT, the recipe would meet contrast requirements. I think we should consider adding this for the MSFT implementation here. @bheston - thoughts on the recipe to use for the contrast requirement implementation?